azext_iot/central/providers/v1/device_provider_v1.py

```diff
@@ -38,7 +36,9 @@ def __init__(self, cmd, app_id: str, token=None):
         self._device_credentials = {}
         self._device_registration_info = {}
 
-    def get_device(self, device_id, central_dns_suffix=CENTRAL_ENDPOINT) -> Device:
+    def get_device(
+        self, device_id, central_dns_suffix=CENTRAL_ENDPOINT,
+    ):
```

Comment:
Could update the return type hint rather than remove it.

Reply:
fixed